changelogs/fragments/190-deprecate-ansible-2.9-2.10.yml

```
@@ -0,0 +1,4 @@
+---
+deprecated_features:
+  - "Support for Ansible 2.9 and ansible-base 2.10 is deprecated, and will be removed in the next major release (community.hashi_vault 3.0.0) next spring.
```

```suggestion
  - "Support for Ansible 2.9 and ansible-base 2.10 is deprecated, and will be removed in the next major release (community.hashi_vault 3.0.0) next spring
```
There's already a period after the upcoming brackets.